src/CacheProvider.php

```diff
@@ -43,7 +42,7 @@ public function allowCache(
         }
         $headerValue = $type;
         if ($maxAge || is_integer($maxAge)) {
-            if (!is_integer($maxAge) && !is_null($maxAge)) {
+            if (!is_integer($maxAge) && $maxAge !== null) {
```

Comment:
Is this line relevant for the new version?

Reply:
if not changed, the static analysis (performed by `composer test`) fails